### ISSUE TITLE
example config.json

### ISSUE THREAD

**Issue by siobhanellis** [efuturetoday/homekit-photon-opener#1]
Is this for Homebridge so you can open/close a door? I'm looking to use Photon with a linear actuator and you have done the coding for me.

If so, can we have an example config.json?

**Comment by efuturetoday** [comment]
Yes you can use this plugin to use a Particle Device with the Cloud to send ring events to HomeKit, open a door with a button or mute the ring with a button. Currently this plugin is meant to be used as a opener with ring, mute and open door. If you just want to open a door you have to modify the code an in order to remove the other services.
